wire-library/wire-compiler/src/main/java/com/squareup/wire/WireCompiler.kt

```diff
@@ -143,66 +136,43 @@ class WireCompiler internal constructor(
       )
     }
 
-    Closer.create().use { closer ->
-      val sources = protoPaths.map { it.toPath() }
-      val directories = directoryPaths(closer, sources)
+    val directories = protoPaths.map { it.toPath() }
```

Comment:
I think this needs to be a `Map<Path, FileSystem>` ?

Reply:
Input paths:

```
/Users/bquenaudon/Projects/okio
/Users/bquenaudon/.m2/repository/protos.jar
```

Output map:
```
FileSystem.SYSTEM to "/Users/bquenaudon/Projects/okio".toPath()
FileSystem.SYSTEM.openZip("/Users/bquenaudon/.m2/repository/protos.jar") to "/".toPath()
```